.github/workflows/amoy_deb_profiles.yml

```diff
@@ -382,10 +382,10 @@ jobs:
         run: ls -ltr packaging/deb/ | grep bor
 
       - name: Release bor Packages
-        uses: softprops/action-gh-release@v2
+        uses: softprops/action-gh-release@da05d552573ad5aba039eaac05058a918a7bf631
```

Comment:
Any reason why we're using a commit hash instead of a stable release?

Reply:
https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions

Reply:
Yeah I see. Not sure tho if we should not largely used actions such as `actions/checkout`, which are referenced everywhere with release versions.
Using commits SHA is inconvenient imho when it comes to tracking and updates, as it's not clear what version is being used, and it makes more complicated for 3rd party security tools to detect if that version is affected by sec vulns.

Reply:
Ok will revert back on this PR and the heimdall PR.

Reply:
I mean it's basically harder to read and maintain (we don’t know which version or features we're using without looking it up). True that we have the risks of backdoor, but I think it's so minimal, and also pinning a commit SHA won't get us with security patches or improvements, unless we manually check for updates and bump the SHA (this can lead to technical debt if forgotten). Not sure, it feels like an overkill for "trusted" actions like actions/checkout, since GitHub maintains them "securely".
Up to you, just my 2 cents.